web/styles/compose.css

```diff
@@ -284,12 +284,12 @@
     align-items: center;
     justify-content: space-between;
     font-size: 15px;
-    line-height: 18px;
+    line-height: 15px;
```

Comment:
I think we want 14px `font-size` to match the rest of the app (we'll change this when changing the overall app font size, I think). And the line-height of 15px here looks bad for the multiline cases; I think it needs to be at least 17px. 

I'm not sure which element in Figma this version corresponds to so I can't say what is intended.

Reply:
The line height prevents the banner from being short enough, because the banner is essentially just the height of the `<p>` element.

Reply:
Good point about the multi-line though. I'll check again if I can do it with just padding, I was actually just guessing why  I did it.

Reply:
On further investigation, the line-height only changes things if there are no buttons, and the height with button seems to match Figma. Not really sure what past-me was doing 🤷 

<img width="1299" alt="image" src="https://user-images.githubusercontent.com/5634097/228147054-f749c706-faec-4b73-9357-e1dfd13065d4.png">